### PR TITLE
Add SLE Updates testsuite for 5.0

### PR DESCRIPTION
**Pull request by Bischoff**
main.tf for new test suite

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+module "base" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-su-50-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles15sp6o", "opensuse155o", "slemicro55o" ]
```
Comment (vandabarata) [inline terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:99]: do we need the `opensuse1550`  image? coudn't see it here :thinking:

Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:99]: for the controller, yes

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf [hunk 2]
```diff
@@ -0,0 +1,223 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/view/Manager/view/Manager-5.0/job/manager-4.3-qe-sle-update"
```
[the remark this answers is not in the document]
Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:4]: oops! thanks

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf (hunk 1 above)
Comment (szachovy) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:99]: I guess it's built-in for controller. _EDIT: accidentially deleted comment_

Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:99]: fixed

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf]: fixed

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base.configuration
+  product_version    = "5.0-released"
+  name               = "srv"
+  image              = "slemicro55o"
+  provider_settings = {
+    mac                = "aa:b2:92:05:00:fd"
+    memory             = 40960
+    vcpu               = 10
```
[the remark this answers is not in the document]
Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:121]: i guess it's inherited from an old copy-paste from BV

I diminished it, thanks

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+    data_pool          = "ssd"
+  }
+
+  main_disk_size        = 1500
+  runtime               = "podman"
+  container_repository  = "registry.suse.de/suse/sle-15-sp6/update/products/manager50/containerfile/suse/manager/5.0/x86_64"
```
[the remark this answers is not in the document]
Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:127]: Can you please tell me which one i should use ?

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+  name               = "srv"
+  image              = "slemicro55o"
+  provider_settings = {
+    mac                = "aa:b2:92:05:00:fd"
+    memory             = 40960
+    vcpu               = 10
+    data_pool          = "ssd"
+  }
+
+  main_disk_size        = 1500
```
[the remark this answers is not in the document]
Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:125]: repository_disk_size is for distinct disks and is used in 4.3

main_disk_size is for a single disk and is used in 5.0

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+  //server_additional_repos
+
+}
+
+module "proxy_containerized" {
+  source             = "./modules/proxy_containerized"
+  base_configuration = module.base.configuration
+  product_version    = "5.0-released"
+  name               = "pxy"
+  image              = "slemicro55o"
```
[the remark this answers is not in the document]
Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:154]: yes, but on the other hand i don't really trust sumaform's default, and it's also documenting the correct value

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf (hunk 4 above)
[the remark this answers is not in the document]
Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf]: trusting you blindly on this one

done

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf (hunk 5 above)
[the remark this answers is not in the document]
Comment (szachovy) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:125]: ok

File: terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf (hunk 6 above)
[the remark this answers is not in the document]
Comment (szachovy) [reply terracumber_config/tf_files/SUSEManager-5.0-SLE-update.tf:154]: ok

**Review by vandabarata — APPROVED** [review]

**Pull request merged**